src/utils/daos_hdlr.c

```diff
@@ -1407,6 +1407,7 @@ cont_create_hdlr(struct cmd_args_s *ap)
 		attr.da_oclass_id = ap->oclass;
 		attr.da_chunk_size = ap->chunk_size;
 		attr.da_props = ap->props;
+		attr.da_mode = ap->mode;
```

Comment:
Since the mode is only used for POSIX, do we want `daos.c` to print an error if a user gives a mode for a non-POSIX container?

Reply:
Hmm. On second thought, it seems that `daos.c` doesn't check against this for other utilities in general. Maybe we don't care?

Reply:
I see you updated this. Thanks!